pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Root.groovy

```diff
@@ -143,11 +143,22 @@ public class Root implements NestedModel, Serializable {
      */
     @Whitelisted
     List<String> getEnvVars() {
-        return environment.collect { k, v ->
+        return environment.findAll{k, v -> !(v instanceof CredentialWrapper)}.collect { k, v ->
             "${k}=${v}"
         }
     }
 
+    @Whitelisted
```

Comment:
No need for `Whitelisted` - ive got that pending PR to remove it everywhere, so let's not add it in new places too. =)